run/helm/snap/templates/daemonset.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: snap
+  namespace: {{ .Values.namespace }}
+  labels:
+    kubernetes.io/cluster-service: "true"
```

Comment:
@andrzej-k I have seen this on kubernetes manifests in the addons section. Does it actually not do anything?